src/test/integration_tests/internal/test_upgrade_scripts.js

```diff
@@ -159,6 +158,8 @@ mocha.describe('test upgrade scripts', async function() {
         const prefix_exists = system_store.data.pools.find(pool => pool.name.startsWith(base));
         assert.strictEqual(exact_removed, undefined);
         assert.strictEqual(prefix_exists, undefined);
+        const exact_added = system_store.data.pools.find(pool => pool.name === pool_name);
```

Comment:
maybe we should add a check here for buckets using the mongo pool to change to the new pool